app/controllers/admin/schedule_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Admin
+  class ScheduleController < Admin::BaseController
```

Comment:
Why did you rename 

```
 app/controllers/admin/schedules_controller.rb 
```
to
```
app/controllers/admin/schedule_controller.rb 
```

http://alexander-clark.com/blog/rails-conventions-singular-or-plural/